API/files/README.md

```diff
@@ -28,7 +28,8 @@ If no `content` is passed, then the path is treated as an empty directory
 ```js
 {
   path: '/tmp/myfile.txt',
-  node: DAGNode
+  hash: 'QmHash', // base58 encoded multihash
```

Comment:
This should not be a base58 encoded multihash. Looking at the future we are introducing cid + ipld soon. That means we have to break this interface at that point to a degree. This should be mulithash buffer/object something like that.